futurenhs.automation/features/foldersManagement.feature

```diff
@@ -100,13 +100,14 @@ Scenario Outline: FNHS69 - Update folder error validation
pddo4PGbpjzV1Ofga9L83HaBWCkHOrn98kQ2iqBggGXRnEmLrws5paf5Li0RF4HOOkmvo0Es9kcn6NJXBDME4QPQdTWUGLXFxMjCVU7NT3OyKgOVYUmtS3AZagofAir3TS0y1XUE1fhIuMjpAaaeMsAeWXdPoi4BfoHgc35zg1ZSgqYUg5TMBCXThegvddd82pauNthAH22RjozawZF0Y0qBVpdxklRIvLE95jodJsf7yPJXitylPUVTQR9slBKzHxQ9cDzVl4nyhjcY3QhZqzJGk1Ks2o611s5BERTHJTO1L9wM0sm2rZrli8WtmsiTfilCFejYNVALaHm12pclYBDG2cJhiAocLNFnLSpKnizJC1dkq2lzVlgmIOEH5RrCiKpcndqETmZEulZo7JmKGk2LXGI1BauAdmxddavm6YwRqROnSBbRnM4i6JAFFWRTqbjckegjCuWYBQTHPmf3LOPy2J2xKKa0ry1vcbn6L63O0zfEsOmyQxxqBDri2AfGcuXaT2NtiYsIDv8VUAjzStvFGfegEk8EQemWjMsYFLDXx0DCM7cZQC31rJCchGT4TGZsFAMZ5xy8aPgDxVcGCsUlBdJmDMx9gtj87Fw1V7MH5OTn6UVE1VqEmWZlpXNLv7cAwN5RoIX3jmmRkuJ1GOVo3tnFmHnOXyakiB1w7LQShZr0N6wMh2bxUNDPEoiOOqVTWKJGpeONuC373bJauYTkNy2oAHPaAgWZ7RIj3ZZ7bqCjkdtrybHJOC8cINQApzL9eUuxtcUwgfeu68tNwbjejD8ezBHDBp3uK6fOZRwauuy3cbMQb9ptjTngO7lKxjkmIzpI69hahcbDvOm6i2QkvNnHtU3tRHrDtW7MZPsic2t9FrYaTeeKPERUDYU7nODhYbwYgoSS1y1kZGCAhijR5jdvrt4jVYV0Y43rGeDrvks78Oup3xXrsjVEE8sUyy02NYSO0vcJ1NdeOWQBCctDKr0wctrucYnViqDr5yZPRyBg2G3Xu3xjMOgVpZPVvG1Vd4E5WMbsT5iBTMEJNKM1AWrJ3zvWNuAocODdB2GVwVO2zR5mWlfAVn16trHSnYWL89isi2QXsvsjsp2oFjrtTS1igwRcrnvYyLMnp4a2pgUKZiTJoGeiCKhDIxSSMnqqH2db9EqRdI1rJ8cPAeHFdBMKMF5wstWBkQWaUWJT2ltE7ydtqgfacS5qf8AgphOcmGDNsff9ICTegTQwYA0UlA4KbI3Uv725pUxGxd7nZ2rqnLuaYlQKjDgcEKvKavTlk0VKYZYaHhlYMjIkIiI9f9vNuLf19W5BWSTDzog336J8WHa4NNYc6RPWn2WnnofIxYeQr3tp3e3GuweuWx4NkxQ1fTtsXdw97hsOIGjpj8JSlnLt3xYlo8X7DX6hzeYaDFBoAOXf2akF0cCtHO6tEphuTarP5pm5HsLdpXzJnJ1cz4PzYK5W0EPo64qCfaUxftFLCmV9s4qVZcdGy7fyvVl6v7X3FJkSjxBTLFfXHw4dtzsHTaaDALYYawfNxfI2R4DJboek0eZiA3vmTy6I9f4hQoNrmKwlibo9FwGKnYKGRqBZT8cwHiFmLjW3WeGwYnivfNI4UsvUTqRaEYRhBjaVH7FIA6mhhfH9zftjzRfO4Ynbu1Y9pPqfgJxneVQiREONjiS40uPGPRoAraWWkTJ6zRnPS9kYw4SOxvrgHwPcPVK3zsmBXZqLk2l5Rm85I69gCacvpykZMMwV7xyOg3FEQEFfes8sVyEu19gchztYC5HABKFK3MvNfiyjPWwd9V4yurBANmJiRL80Lj20qgNYUslPnYWVarkdnDhosniZ9m9MkCWgvekIjAaSAsUOBHGBK7OPLolufQlnxmljLb7y3iqniUTKWeGbqfDy1bgrPXRfiupZy8Kb9c3GO6p66nUwQLXkFlhXFP7hvx9vUND55nmcJ32m8uMcYTrwoanEP7gpYsJoWzbJXBfZZsTJjwmeDfAH5rYwKIUaGD9Xuox1bbFaoQoztHj5sF7EffMkwysBhU8tstJi6SgnGDAWN1T3sGJgzj62XcOWQnzsNZuvm0z74LZEuXgi4WTikkQujuxn8H0Ap7nVy7ABPfkGN5595JjIsh9WupYqjtTibOxLT4sLoewgvIzNpSTOoItg1k5Ho4BKuhgus0I7TedMtEaLJJA18gKXVC8PEjpXD2ncnbtV7vJH1Q8Dw55rnTjLsawAURMAgoJfySDc0agxFP3uVFUkt7HR5QZqRJw6W21iO4Emj6FltlAbeIMdyvgMHjORB8QPWghXQne0eC8yiBy92oJvlEcexJzmJDUPjz0PBS7LNVRTzHpgpaoCbw4JDWJ1Q48Y8CWyA7U3HPwpRdCqx4PJNhF8C8Jm8zLp6oib0TX7TgRSjMI8LzPOwCx9onywmPm53olifsorwfsafRFBcLsSo0YiZMrHY2NfqtB8BqJJyL5unIDvV36cuhoMNONpdFFnAs8s6imfno3iUdTZL0Iihsky9UycKB8zcKUWKGnUgJJOUUw8rENOEOjrYIXxR1ytYKLfuTI7nlkmsfLKBKn8OeOzDkI98iJYOh4pmmQ1em0Sa8vARw203A7clonmQRbUsXRy76GghO6eKgeTZjng8T8sp70Vqz12LXLszrFa2NjznvxOBS2i5DsV6HHeg9xEQgoohpcOyEKV6VRPaF92FlDDDp2gWXcZULhLrl45lKZujMbaxDNNQuWfIzmidLqrYmXnFempbtE2lzdGXofXToPQZDEqDcuiuWZbUm1PVIx1oMunx7weq17qwzcbBuRebAjnJkRBRz6EfoN0cbwZaxEybtlZCCg7eINPYXJeC2yIVkW2nMPSpHpxWL2A3OkLJEysp5DSPpNZ879ZmYXyy1Xkvv19tDUUBJ3Gy5UQYoXUpRnbMX4H9EeHzVOVb79r7zf4neONZ971I8uKHuXGhkNhHdrkl | Enter 4000 or fewer characters |
```

Comment:
these long strings hurt the readability somewhat, can i make the suggestion to replace them when you get time with something like [4001 random chars] in the feature files and have them generated when the test is run by a rng